### PR TITLE
Add notes and links about merchant refunds

### LINKED ISSUE #952: FAQ question regarding merchant refunds
This is a very common issue in the chat: "I paid to xyz merchant and I need a refund, etc". Example from chat [here](https://chat.btcpayserver.org/btcpayserver/pl/c9rxrkefofyn5xinyee1h5b1sr). The issue is addressed in the following FAQ question: https://docs.btcpayserver.org/FAQ/FAQ-General/#what-if-i-have-a-problem-with-a-paid-invoice 

This question does answer the topic quite well but the visibility needs to be improved. This FAQ question should come up in the search for Refunds. It should also probably be linked in the [Refunds doc](https://docs.btcpayserver.org/Refund/#refunds) for any user who ends up there by mistake looking for this information.

### PR DESCRIPTION
**Pull request by PresidentNick**
Does this look like it'll resolve #952?

It looks like my IDE removed a bunch of spaces too, I can remove that from the commit if needed.

### CODE REVIEW

**Review by Zaxounette — COMMENTED** [review]
The only nitpick I have is that the warning could be changes to a tip.
We don't want to abuse the orange warnings too much.

Other than that, i'll leave last dibs to @britttttk on if this satisfactory for her.

Once my nitpick is changes, ACK for me.

Thanks for your help!

**Review by Zaxounette — APPROVED** [review]

**Comment by PresidentNick** [comment]
I fixed a minor grammar issue I found in the tip I wrote with the previous force-push

File: docs/FAQ/FAQ-General.md [hunk 1]
```diff
@@ -121,8 +127,12 @@ Users may try to pay an invoice from an exchange or other custodial service wher
 
 ## What if I have a problem with a paid invoice?
 
+:::tip
+To request a refund from a merchant, please contact the merchant directly! BTCPay Server has no relationship to the merchant you purchased the product or service from.
```
Comment (britttttk) [inline docs/FAQ/FAQ-General.md:131]:
```suggestion
To request a refund from a merchant, you must contact the merchant directly! BTCPay Server has no relationship to the merchant you purchased the product or service from.
```

File: docs/Refund.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Refunds
 
+:::tip
+If you are looking for information about claiming a refund from a merchant, please refer to this [FAQ](./FAQ/FAQ-General.md#what-if-i-have-a-problem-with-a-paid-invoice)
```
Comment (britttttk) [inline docs/Refund.md:4]:
```suggestion
If you are looking for information about requesting a refund from a merchant, please refer to this [FAQ](./FAQ/FAQ-General.md#what-if-i-have-a-problem-with-a-paid-invoice)
```

**Review by britttttk — APPROVED** [review]

**Comment by britttttk** [comment]
> This build failed on the Linkcheck, but there is no output on which link is broken

image

link seems fine though :)

**Comment by PresidentNick** [comment]
@britttttk I was expecting the error at the bottom of the log, not in the middle, woops! I wonder why linkchecker thinks it's broken

**Comment by PresidentNick** [comment]
Oh cause it returns 403

**Review by pavlenex — APPROVED** [review]

**Pull request merged**